packages/maskbook/src/database/Persona/Persona.db.ts

```diff
@@ -515,9 +508,14 @@ export interface PersonaRecord {
      */
     uninitialized?: boolean
 }
-type ProfileRecordDB = Omit<ProfileRecord, 'identifier' | 'hasPrivateKey' | 'linkedPersona'> & {
+type ProfileRecordDB = Omit<
+    ProfileRecord,
+    'identifier' | 'hasPrivateKey' | 'linkedPersona' | 'relatedPersonaIdentifier' | 'favorite'
+> & {
     identifier: string
     network: string
+    relatedPersonaIdentifier?: string
```

Comment:
It seems like this model doesn't allow a Profile to be bonded with multiple personas.